lib/hyacinth/adapters/external_identifier_adapter/abstract.rb

```diff
@@ -4,7 +4,9 @@ module Hyacinth
   module Adapters
     module ExternalIdentifierAdapter
       class Abstract
-        def initialize(adapter_config = {}); end
+        def initialize(adapter_config = {})
+          @default_target_url_template = adapter_config[:default_target_url_template]
```

Comment:
Could you please add this `default_target_url_template` param (with example value) to the datacite.README.md file?

Reply:
Though you may want to rebase first to get the other recent changes to that file.

Reply:
documented!